<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>ReverberationParameters</class>
 <widget class="QDialog" name="ReverberationParameters">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>440</width>
    <height>538</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <layout class="QGridLayout" name="gridLayout">
   <item row="0" column="0">
    <layout class="QVBoxLayout" name="verticalLayout_2">
     <item>
      <layout class="QHBoxLayout" name="horizontalLayout">
       <item>
        <widget class="QLabel" name="labelParam_1">
         <property name="text">
          <string>labelParam_1</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QDoubleSpinBox" name="param1Box">
         <property name="maximum">
          <double>60000.000000000000000</double>
         </property>
        </widget>
       </item>
      </layout>
     </item>
     <item>
      <layout class="QHBoxLayout" name="horizontalLayout_2">
       <item>
        <widget class="QLabel" name="labelParam_2">
         <property name="text">
          <string>labelParam_2</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QDoubleSpinBox" name="param2Box">
         <property name="maximum">
          <double>60000.000000000000000</double>
         </property>
        </widget>
       </item>
      </layout>
     </item>
     <item>
      <layout class="QHBoxLayout" name="horizontalLayout_3">
       <item>
        <widget class="QLabel" name="labelParam_3">
         <property name="text">
          <string>labelParam_3</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QDoubleSpinBox" name="param3Box">
         <property name="maximum">
          <double>60000.000000000000000</double>
         </property>
        </widget>
       </item>
      </layout>
     </item>
    </layout>
   </item>
   <item row="1" column="0">
    <layout class="QVBoxLayout" name="verticalLayout_3">
     <item>
      <layout class="QHBoxLayout" name="horizontalLayout_4">
       <item>
        <widget class="QLabel" name="lablelChannel_1">
         <property name="text">
          <string>lablelChannel_1</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QSpinBox" name="channel1Box">
         <property name="minimum">
          <number>1</number>
         </property>
         <property name="maximum">
          <number>30</number>
         </property>
        </widget>
       </item>
      </layout>
     </item>
     <item>
      <layout class="QHBoxLayout" name="horizontalLayout_5">
       <item>
        <widget class="QLabel" name="lablelChannel_2">
         <property name="text">
          <string>lablelChannel_2</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QSpinBox" name="channel2Box">
         <property name="minimum">
          <number>1</number>
         </property>
         <property name="maximum">
          <number>30</number>
         </property>
        </widget>
       </item>
      </layout>
     </item>
    </layout>
   </item>
   <item row="2" column="0">
    <layout class="QVBoxLayout" name="verticalLayout">
     <item>
      <widget class="QCheckBox" name="surfCheckBox">
       <property name="text">
        <string>Певерхностная реверберация</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QCheckBox" name="surroundCheckBox">
       <property name="text">
        <string>Объемная реверберация</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QCheckBox" name="botCheckBox">
       <property name="text">
        <string>Донная реверберация</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QCheckBox" name="sumCheckBox">
       <property name="text">
        <string>Суммарная реверберация</string>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item row="3" column="0">
    <layout class="QHBoxLayout" name="horizontalLayout_7">
     <item>
      <widget class="QLabel" name="label">
       <property name="text">
        <string>Количество точек расчета</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QSpinBox" name="numDotBox"/>
     </item>
    </layout>
   </item>
   <item row="4" column="0">
    <layout class="QHBoxLayout" name="horizontalLayout_6">
     <item>
      <widget class="QPushButton" name="pushButtonCompute">
       <property name="text">
        <string>Расcчитать</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="pushButtonSave">
       <property name="text">
        <string>Сохранить</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="pushButtonCancel">
       <property name="text">
        <string>Отмена</string>
       </property>
      </widget>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
